### DISCUSSION TITLE
Update Email Address For Updraft.cyfrin.io

### DISCUSSION THREAD

**Discussion by kmatthewsio** [Cyfrin/foundry-full-course-cu#1686] [General]
I used the e-mail service provider skiff.com which is retiring. I've updated my github account to a new email but I can't seem to change it in Updraft.cyfrin.io

**Comment by EngrPips** [comment]
Hello @2krypto2quit, I will advise you to report this on the official discord for cyfrin `https://discord.gg/RXvvJBwX` as that is the more appropriate channel to report stuff of this nature.
